### ISSUE TITLE
Waterfall-based?

### ISSUE THREAD

**Issue by gp-Airee** [Leymooo/BungeeCord#42]
Could these changes be made against Waterfall or maybe done via patches so it can be applied among other modifications of Bungee?

**Comment by callmesensey** [comment]
Why not do it? @Leymooo

**Comment by Leymooo** [comment]
I will rewrite botfilter, because for now it has an old code base which is bad, this is why i do not rebase BF to waterfall. There is no eta when i will start rewrite, but I will try not to delay for too long a period.

**Comment by callmesensey** [comment]
Ну давай, ждем-с. А когда будешь переписывать, на WF тоже не планируешь основывать?

**Comment by Leymooo** [comment]
Планирую.

**Comment by gp-Airee** [comment]
Thank you. Also worth noting that waterfall uses a patch system to add their patches, perhaps botfilter can be done this way?

Спасибо. Также стоит отметить, что водопад использует систему патчей для добавления своих патчей, возможно, это можно сделать с помощью botfilter?

**Comment by DoNotSpamPls** [comment]
Hey, is there any progress on the rewrite? 👀

**Comment by y6x2digc** [comment]
just download this diff https://github.com/SpigotMC/BungeeCord/compare/master...Leymooo:master.diff and apply it to Waterfall

**Comment by gp-Airee** [comment]
Don't know how

**Comment by y6x2digc** [comment]
@gp-Airee
```
$ wget -O a.diff https://github.com/SpigotMC/BungeeCord/compare/master...Leymooo:master.diff
$ cd [Waterfall's dir]
$ patch -p1 < [path to a.diff]
```
and solve conflicts

**Comment by y6x2digc** [comment]
@Leymooo I don’t think the code base is that bad

**Comment by NoJokeFNA** [comment]
@Leymooo Sup, any news about Waterfall based fork? Have you tried this guy's diff command yet?

**Comment by NotPhasmatos** [comment]
Any news?

**Comment by NoJokeFNA** [comment]
Yes, Leymooo won't spend any time migrating it to Waterfall based Fork.
